Example/Backpack/View Controllers/SnackbarSelectorViewController.swift

```diff
@@ -35,6 +35,9 @@ class SnackBarSelectorViewController: UITableViewController {
         setupDestinationController(destinationController: destinationController, segue: segue)
     }
 
+    // Cyclomatic complexity might be high in this function but it's actually really
```

Comment:
This comment seems a bit yagni 🤷‍♂️

Reply:
I feel like it's a good idea to include a comment when disabling lint rules, that's why I added it